CustomerRegister/Repositories/CustomerService.cs

```diff
@@ -0,0 +1,73 @@
+﻿
+
+using CustomerRegister.Repositories.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CustomerRegister
+{
+    public class CustomerService : ICustomerService
+
+    {
+        private readonly List<CustomerEntity> _customers = new();
+
+        public bool Exists(CustomerEntity selectedCustomer)
+        {
+            var response = _customers.Any(customer => customer.Cpf == selectedCustomer.Cpf || customer.Email == selectedCustomer.Email) && (selectedCustomer.Id != selectedCustomer.Id);
+            return response;
+        }
+
+        public bool DuplicatedRegister(CustomerEntity selectedCustomer)
+        {
+            if(_customers.Any(customer => customer.Cpf == selectedCustomer.Cpf || customer.Email == selectedCustomer.Email))
+                {
+                return true;
+                }
+            return false;
+        }
+
+        public bool AddCustomer(CustomerEntity customer)
```

Comment:
```suggestion
        public int AddCustomer(CustomerEntity customer)
```
Já que você optou por configurar um tipo no retorno. Deveria retornar o id do customer criado.

Reply:
Feito